pytest_splunk_addon/helmut_lib/SearchUtil.py

```diff
@@ -162,6 +162,13 @@ def checkQueryCountIsZero(self, query, max_time=120):
             self.logger.debug("Count of results is > 0, it is:%d", result_count)
             return False, job.get_results()
 
+    def doSearch(self, query, max_time=20):
+        self.logger.debug("query is %s", query)
+
+        job = self.jobs.create(query, auto_finalize_ec=200, max_time=max_time)
+        job.wait(max_time)
+        return job.get_results()
```

Comment:
We should use exception to signal if function output is not complete/malformed.

Reply:
added try/except to the code which is getting search results